test/helpers.js

```diff
@@ -33,7 +33,7 @@ export const fixtures = fs
   .map(file => `${__dirname}/fixtures/${file}`)
   .filter(f => !f.endsWith('.json'));
 
-export function babelRunFixture(fixture) {
+export function babelRunFixture(fixture, only) {
```

Comment:
there isn't a good way to filter down tests since i'm generating them from files, didn't seem any harm in leaving it, at least until something is better